src/profile-logic/profile-data.js

```diff
@@ -911,7 +914,21 @@ export function getThreadProcessDetails(thread: Thread): string {
 
 export function getEmptyProfile(): Profile {
   return {
-    meta: { interval: 1 },
+    meta: {
+      interval: 1,
+      startTime: 0,
+      shutdownTime: 1000,
+      abi: '',
+      misc: '',
+      oscpu: '',
+      platform: '',
+      processType: 0,
+      product: 'Firefox',
+      stackwalk: 0,
+      toolkit: '',
+      version: GECKO_PROFILE_VERSION,
+      preprocessedProfileVersion: PROCESSED_PROFILE_VERSION,
+    },
```

Comment:
Because I improved the flow typing for our profiles, I also need to return a proper structure.